pybamm/models/submodels/thermal/base_thermal.py

```diff
@@ -175,15 +175,25 @@ def _get_standard_coupled_variables(self, variables):
             Q_rev_n, pybamm.FullBroadcast(0, "separator", "current collector"), Q_rev_p
         )
 
+        # Contact resistance heating Q_cr
+        # TODO check that the volume is the correct volume for contact resistance
+        if self.options["contact resistance"] == "true":
+                I = variables["Current [A]"]
+                V = param.V_cell
+                Q_cr = I**2 * param.R_contact / V
+            else:
```

Comment:
`else` is not aligned with the `if` statement so the tests won't run

Reply:
you can install the "ruff" linter in vscode to highlight things like this

Reply:
another way is to do `pip install pre-commit` and then `pre-commit install` which will block changes that don't pass from being committed

Reply:
Thanks. I updated the code to have the proper syntax.